synapseclient/models/file.py

```diff
@@ -660,15 +661,16 @@ async def _load_local_md5(self, syn: "Synapse") -> None:
         """Load the MD5 of the file if it's a local file and we have not already loaded
         it."""
         if not self.content_md5 and self.path and os.path.isfile(self.path):
-            self.content_md5 = await utils.md5_for_file_multiprocessing(
-                filename=self.path,
-                process_pool_executor=syn._get_process_pool_executor(
-                    asyncio_event_loop=asyncio.get_running_loop()
-                ),
-                md5_semaphore=syn._get_md5_semaphore(
-                    asyncio_event_loop=asyncio.get_running_loop()
-                ),
-            )
+            # self.content_md5 = await utils.md5_for_file_multiprocessing(
+            #     filename=self.path,
+            #     process_pool_executor=syn._get_process_pool_executor(
+            #         asyncio_event_loop=asyncio.get_running_loop()
+            #     ),
+            #     md5_semaphore=syn._get_md5_semaphore(
+            #         asyncio_event_loop=asyncio.get_running_loop()
+            #     ),
+            # )
+            self.content_md5 = utils.md5_for_file_hex(filename=self.path)
```

Comment:
I want to re-visit the usage of using multi-processing for this. There is a tipping point somewhere that the time to start up a new process actually takes longer than just running this in the entrant python process.

Also, I was using `top` to look at resource usage and overall system usage is significant lower when not running this in a second process.

Reply:
I created https://sagebionetworks.jira.com/browse/SYNPY-1489 to look back at this